nemo_text_processing/text_normalization/rw/taggers/cardinal.py

```diff
@@ -0,0 +1,319 @@
+# Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+# Copyright (c) 2024, DIGITAL UMUGANDA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pynini
+from pynini.lib import pynutil
+import string
+from nemo_text_processing.text_normalization.en.graph_utils import GraphFst,NEMO_CHAR,insert_space
+from nemo_text_processing.text_normalization.rw.utils import get_abs_path
+
+def apply_fst(text, fst):
```

Comment:
this method is unnecessary. the normalize class takes care of this with additional preprocessing. Please remove.